src/modules/bricks/volume-bricks/volume-bricks.html

```diff
@@ -107,7 +107,7 @@ <h5 ng-if="vm.subVolumeList.length !== 0">{{vm.totalBrick}}
                         <div class="list-group list-view-pf list-view-pf-view " ng-repeat="brick in subVolume.bricks | orderBy: 'fqdn'">
                             <div class="list-group-item">
                                 <div class="list-view-pf-actions">
-                                    <button class="btn btn-default" ng-click="vm.redirectToGrafana(brick, $event)">Launch Dashboard</button>
+                                    <button class="btn btn-default" ng-click="vm.redirectToGrafana(brick, $event)">Launch</button>
```

Comment:
@cloudbehl, according to https://github.com/Tendrl/ui/issues/651, 'Launch Dashboard' -> 'Dashboard'